README.md

```diff
@@ -0,0 +1,17 @@
+# otm-website-tests
+
+An automated test suite for the OpenTreeMap website.
+
+## Usage
+
+To execute the test suite, first install [Leiningen](http://leiningen.org/), then execute the following command:
```

Comment:
Addressed in https://github.com/OpenTreeMap/otm-website-tests/commit/ab1a12bd2f2cbdc73fdb6fa003b4999249ee5ba5.